make-prs.py

```diff
@@ -0,0 +1,251 @@
+import argparse
```

Comment:
It would be nice if there was a module-level docstring that briefly explained what the file does (it could even say to read the HACKING.md file for more details).

Similarly, it would be nice if each function (except main) had a short (e.g. one-sentence) docstring.

Reply:
I added a module-level docstring with more information. Gently push back on the function docstrings because the functions are well-named enough, and we probably won't be creating documentation based on the docstring for this script.